qemu/tests/cfg/vsock.cfg

```
@@ -12,5 +12,8 @@
             vsocks = ''
             addr_pattern = '\d\d:\d\d\.\d'
             device_pattern = 'Communication controller: Red Hat.* Device'
+            q35:
+              RHEL.8.3, RHEL.8.4:
+                device_pattern = 'Communication controller: Red Hat.* Virtio socket'
```

This way you will always need to append new version when it comes out.
```suggestion
            device_pattern = 'Communication controller: Red Hat.* Device'
            q35:
                device_pattern = 'Communication controller: Red Hat.* Virtio socket'
                RHEL.7, RHEL.8.1, RHEL.8.2:
                    device_pattern = 'Communication controller: Red Hat.* Device'
```

---

Updated. Thank.

---

python3 ConfigTest.py --guestname=RHEL.8.4.0 --testcase=vsock_test.hotplug --clone=no --machine=q35
(1/1) Host_RHEL.m8.u4.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.vsock_test.hotplug.q35: PASS (169.00 s)

python3 ConfigTest.py --guestname=RHEL.8.2.1 --testcase=vsock_test.hotplug --clone=no --machine=q35
(1/1) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.1.x86_64.io-github-autotest-qemu.vsock_test.hotplug.q35: PASS (161.31 s)